src/ImmersedBoundaries/grid_fitted_immersed_boundary.jl

```diff
@@ -164,3 +164,5 @@ end
 end
 
 mask_immersed_velocities!(U, arch, grid::GFIBG) = Tuple(mask_immersed_field!(q) for q in U)
+
+mask_immersed_tracers!(tracers, arch, grid::GFIBG) = Tuple(mask_immersed_field!(c) for c in tracers)
```

Comment:
Huh, I'm not sure why we have `mask_immersed_velocities!`. Note that `mask_immersed_tracers!` is identical to the function for velocities, so maybe we don't need both...